Simulator/Test/test_sun_model.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import os, sys, inspect
+import math
+import numpy as np
+import julian
+
+# add current folder to the path
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+groundtruth_dir = os.path.dirname(currentdir)
+sys.path.insert(0, groundtruth_dir)
+
+import sun_model
+
+
+# angle between two vectors
+
+def test_sun_position_ECI():
+
+	mjd = 54000
+	dt = julian.from_jd(mjd, fmt='mjd')
+
+	a = sun_model.approx_sun_position_ECI(mjd)
+	b = sun_model.sun_position_ECI(dt)
+	angle_diff = math.acos(np.dot(a,b)/(np.linalg.norm(a)*np.linalg.norm(b)))*180/math.pi
+
+	assert angle_diff < 5
```

Comment:
I just tested this starting on different dates and we exceed 5 degrees. Is there a larger tolerance we should be using, or does this mean the approximation isn't good enough for our purposes? @GathererA

Reply:
weird, shouldn't be that significant, I wish I had time to look into this. What are y'all thinking? Sorry I got to this so late.

Reply:
Did you compare to JPL ephemerides?

Reply:
This was resolved by using astropy's built in GCRS to ECI conversion.